### ISSUE TITLE
[Accessibility] Page Load Progress beeps are broken in 1.2

### ISSUE THREAD

**Issue by mozillamobilebot** [mozilla-mobile/firefox-ios#4985]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1225749

The progress beeps that we used to play up until Firefox for iOS 1.1 got broken in the 1.2 update. To reproduce:

1. Turn on VoiceOver.
2. Load any page, e. g. from your home page.

Expected: While page loads, progress beeps should be heard.
Actual: Only the final ascending tone played by VoiceOver is heard that indicates a change in view layout. Our own progress beeps, the "heartbeat" and the "ascending beeps" are broken, do not play.

**Comment by SimonBasca** [comment]
The page beeps when starting to load and also beeps when it is fully loaded. This the exact behavior as Safari.
